rel/files/app.config

```diff
@@ -43,11 +44,32 @@
          },
          formatter => {mongoose_flatlog_formatter, #{
            map_depth => 3,
-           term_depth => 50,
-           template_name => mongoose_default_log_template
+           term_depth => 50
          }}
-        }
-    }
+    }},
+
+    %% JSON file log handler:
+    {handler, disk_json_log, logger_disk_log_h, #{
+         level => all,
+         config => #{
+           file => "{{mongooseim_log_dir}}/mongooseim.json",
+           type => wrap,
+           max_no_files => 5,
+           max_no_bytes => 2097152,
+           sync_mode_qlen => 2000, % If sync_mode_qlen is set to the same value as drop_mode_qlen,
+           drop_mode_qlen => 2000, % synchronous mode is disabled. That is, the handler always runs
+           flush_qlen => 5000,     % in asynchronous mode, unless dropping or flushing is invoked.
+           overload_kill_enable => true
+           % Documentation about Overload protection, together with default values, can be found here:
+           % http://erlang.org/doc/apps/kernel/logger_chapter.html#protecting-the-handler-from-overload
+         },
+         formatter => {mongoose_json_formatter, #{
+           format_depth => 10,
+           format_chars_limit => 3000,
+           depth => 10
+         }}
+    }}
+
```

Comment:
Because there is no _easy_ way to enable it in Docker, if needed. And we still need to test both of them on dev machines.

Longer it takes to configure - the higher change we loose the interest of a new user.
It's useful for discoverability of features, people who care about performance would not use the default config most likely.

But we can add info about the fact that one handler could be disabled, if not needed, for better performance.

Reply:
**_Please do add information that there are two handlers enabled by default and disabling one of them will most likely improve the performance if needed_**